doc/man1/flux-start.rst

```diff
@@ -74,6 +74,12 @@ OPTIONS
    is triggered upon exit of any broker, and the flux-start exit code is the
    highest exit code of all brokers.  Default: ``any``.
 
+**--test-start-mode**\ =\ *MODE*
+   Set the start mode.  If set to ``all``, all brokers are started immediately.
+   If set to ``leader``, only the leader is started.  Hint: in ``leader`` mode,
+   use ``--setattr=broker.quorum=0`` to let the initial program start before
+   the other brokers are online.  Default: ``all``.
```

Comment:
Question: should `--test-start-mode=leader` just add `--setattr=broker.quorum=0` to the broker command line by default?

Reply:
I'm hesitant to go there just yet.  It would be more user friendly, but since tests may need to tune the quorum for other reasons (e.g. in other start modes), it feels like it should just be fully under control of the tests.

Could we table that for now?

Reply:
Yes, it was more of a question not a request for a change.

Reply:
OK, thanks.  I'll go ahead and set MWP and rebase the other PR.